github/acls/ansible-collections/kubernetes.core.config

```diff
@@ -0,0 +1,12 @@
+---
+# NOTE(pabelanger): See https://developer.github.com/v3/repos/branches/#parameters-1
```

Comment:
Can you remove this file for now. If we merge this, we delegate to Zuul the configuration of the repository. Since we've got already a functional CI place, it's better to delay this one.